account_ecotax/models/account_tax.py

```diff
@@ -0,0 +1,32 @@
+# © 2014-2024 Akretion (http://www.akretion.com)
+#   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    is_ecotax = fields.Boolean(
+        "Ecotax",
+        help="Warning : To include Ecotax "
+        "in the VAT tax check this :\n"
+        '1: check  "included in base amount "\n'
+        "2: The Ecotax sequence must be less then "
+        "VAT tax (in sale and purchase)",
+    )
```

Comment:
Maybe we may rename is_ecotax to ecotax as fields Selection (('fixed', 'Fixed'), ('weight', 'Weight')] because it can lead to misconfiguration

cc @mourad-ehm @yankinmax    ?

![image](https://github.com/OCA/account-fiscal-rule/assets/1853434/3c8fc75b-82da-406a-a76c-e88a1de87081)

Reply:
`is_ecotax` and `ecotax_type` are different fields, so no misconfiguration here.
Taxes are filtered by `is_ecotax` in computations.
I don't see issue here actually. But, I can miss something of course

Reply:
You're they are different fields but they might the same with a selection field in mixin.

Here ecotax computing fails because product  `_compute_product_ecotax` method search for a fixed ecotax amount and sale order apply 'ecotaxe poids' , then ecotax is 0 on sale order line because there is only fixed ecotax amount not computed